sql/driver/driver_it_test.go

```diff
@@ -417,6 +420,40 @@ func TestSetSSLConfig(t *testing.T) {
 	}
 }
 
+func TestConcurrentQueries(t *testing.T) {
+	it.SkipIf(t, "hz < 5.0")
+	it.SQLTester(t, func(t *testing.T, client *hz.Client, config *hz.Config, m *hz.Map, mapName string) {
+		db := driver.Open(*config)
+		defer db.Close()
+		q := fmt.Sprintf(`
+			CREATE MAPPING "%s"
+			TYPE IMAP
+			OPTIONS (
+				'keyFormat' = 'bigint',
+				'valueFormat' = 'varchar'
+			)
+		`, mapName)
+		t.Logf("Query: %s", q)
+		it.MustValue(db.Exec(q))
+		it.Must(m.Set(context.TODO(), 1, "foo"))
+		const cnt = 1000
+		wg := &sync.WaitGroup{}
+		wg.Add(cnt)
+		for i := 0; i < cnt; i++ {
+			go func(wg *sync.WaitGroup) {
+				var k int64
+				var v string
+				res := db.QueryRow(fmt.Sprintf(`SELECT __key, this from "%s" LIMIT 1`, mapName))
+				if err := res.Scan(&k, &v); err != nil {
+					panic(err)
+				}
```

Comment:
We can assert "k" and "v" here

Reply:
We could do that, but we already test the scanning of correct values  in other tests. This one just makes sure that `QueryRow` can run concurrently without panics.